packages/css/src/components/image-slider/image-slider.scss

```diff
@@ -47,6 +48,10 @@
   grid-column: 1/-1;
   grid-row: 1;
   justify-content: space-between;
+
+  @media (pointer: coarse) and (max-width: $ams-breakpoint-medium) {
+    display: none;
+  }
```

Comment:
Interesting find. Something to remember.

Reply:
The breakpoint is something we don't use elsewhere, we use `min-width`, but it saves a bunch of lines.

Reply:
Is this something I need to resolve?